### ISSUE TITLE
Event for right click?

### ISSUE THREAD

**Issue by hotdog1987** [valor-software/ng2-tree#243]
Is there an event for right click on node? like (NodeRightClicked) ?

**Comment by hotdog1987** [comment]
Yes but is there a way to know if user has right clicked on a node to see the menu? It renders an element 'node-menu' and using angular's elementref to search for element exists is not solving my purpose. I need to perform api call when user right clicks on a node.

**Comment by rychkog** [comment]
@hotdog1987 I think we can publish an event on that action - will bring this in the next release.

**Comment by hotdog1987** [comment]
@rychkog Thank you so much for your attention. Any estimate on this please?
